src/app/pages/movie-details/movie-details.component.html

```diff
@@ -0,0 +1,54 @@
+<ng-container
+  *ngIf="
+    { state: (isLoading$ | async) === false } as isLoading;
+    else preloader
+  ">
+  <ng-container *ngIf="details$ | async as details">
+    <ng-container *ngIf="isLoading.state; else preloader">
```

Comment:
Why don't use this syntax `*ngIf="details$ | async as details; else preloader"`?

Reply:
This syntax won't allow to show preloader when we are moving from one movie detailed page to another, as details will have data from previous movie before new response will take place.

Reply:
Yes, unnecessary else statement in first ngIf. But only removing it didn't helped because for the first load ngIf with `details$ | async` prevented preloder from showing, as a result added additional `else preloader` condition to ngIf with `details$ | async`.  Now user will see preloader both for the first load and when movie changes.